Sources/ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -31,6 +31,12 @@ public final class SQLiteNormalizedCache {
     try self.createTableIfNeeded()
   }
 
+  public init(db: Connection, shouldVacuumOnClear: Bool = false) throws {
+    self.shouldVacuumOnClear = shouldVacuumOnClear
+    self.db = db
+    try self.createTableIfNeeded()
+  }
```

Comment:
This seems like a reasonable addition in terms of allowing people to use SQLCipher or other extensions to SQLite.swift, though, so I do think we should keep this. Can you add some documentation to this initializer please?